sr_finger_mount/launch/sr_finger_mount_launch.launch

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<launch>
+    <arg name="side" default="rh"/>
+    <arg name="fingers" default="ff"/>
+    <node pkg="sr_finger_mount" type="sr_finger_mount.py" name="SrFingerMount" output="screen">
+        <param name="/fingers" value="$(arg fingers)"/>
```

Comment:
please pass array here instead of a string with commas. You can find and example of how to do that here:
https://github.com/shadow-robot/sr_interface/blob/noetic-devel/sr_moveit_hand_config/launch/moveit_planning_and_execution.launch#L40

https://github.com/shadow-robot/common_resources/blob/noetic-devel/sr_utilities_common/src/sr_utilities_common/conditional_delayed_rostool.py#L99

Reply:
The fingers to be used will be probably replaced in the source code itself as it will depend on the hand type (5/3 fingers)